app/matplotlib.ipynb

```diff
@@ -159,20 +207,35 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
-   "outputs": [],
    "source": [
     "data = np.random.normal(0, 1, 1000)\n",
+    "plt.hist(data, edgecolor='black', alpha=0.7)\n",
     "\n",
-    "# write your code here\n"
+    "plt.title(\"Histogram of random data\")\n",
+    "plt.xlabel(\"Value\")\n",
+    "plt.ylabel(\"Frequency\")\n",
+    "plt.show()"
```

Comment:
In Task 4, you need to adjust the bin size to make them not so wide. You can do this by specifying the `bins` parameter in `plt.hist()`, e.g., `plt.hist(data, bins=30, edgecolor='black', alpha=0.7)`.